package/src/kustoMode.ts

```diff
@@ -43,8 +57,14 @@ export function setupMode(defaults: LanguageServiceDefaultsImpl, monacoInstance)
             (worker) =>
                 ({
                     ...worker,
-                    setSchema: (schema) => augmentedSetSchema(schema, worker),
+                    setSchema: (schema) => {
+                        getKustoWorkerTokenParsing().then(worker => worker(first, ...more).then(wor => augmentedSetSchema(schema, wor)));
+                        augmentedSetSchema(schema, worker)
+                    },
                     setSchemaFromShowSchema: (schema, connection, database, globalParameters?: ScalarParameter[]) => {
+                        getKustoWorkerTokenParsing().then(worker => worker(first, ...more).then(wor => wor.normalizeSchema(schema, connection, database)
```

Comment:
.normalizeSchema should be done once (not sure if it's in the worker/main  context )

Reply:
This is happening once for each worker, no?

Reply:
nop, it translation of show schema result to internal model